### PR TITLE
Update indicator description

### PR DESCRIPTION
**Pull request by nicolaskruchten**

### CODE REVIEW

**Comment by antoinerg** [comment]
Thanks very much @nicolaskruchten for the clear description. I am bit surprised I forgot to put it in myself :open_mouth: 

:dancer: :dancer: :dancer:

File: src/traces/indicator/index.js [hunk 1]
```diff
@@ -24,7 +24,11 @@ module.exports = {
 
     meta: {
         description: [
-            'TODO: add description'
+            'An indicator is used to visualize single a `value` along with some',
```
Comment (antoinerg) [inline src/traces/indicator/index.js]: `to visualize single a` :arrow_right: `to visualize a single`

Comment (nicolaskruchten) [reply src/traces/indicator/index.js]: fixed!

**Comment by antoinerg** [comment]
Good to merge once tests passed :)

:dancer:

**Pull request merged**